monitor-agent/deploy.sh

```
@@ -0,0 +1,159 @@
+        docker push $DOCKER_REGISTRY/$DOCKER_IMAGE:$VERSION
+        ;;
+        
+    deploy)
+        log_info "Deploying to $REGION ($SERVER)..."
+        
+        # Build and push first
+        $0 $REGION build
+        $0 $REGION push
+        
+        # Deploy to server
+        ssh $SERVER << EOF
```

Shellcheck (0.10.0)

[warning] Quote 'EOF' to make here document expansions happen on the server side rather than on the client.

(SC2087)